brave/brave-core
# fix(wallet): Move Portfolio Settings to Wallet Settings Menu

## Description 
- Moves the `Portfolio` settings to the `Wallet` settings menu
- Replaces the `Portfolio` settings button with a `Plus` button to open `Edit Visible Assets` modal


Resolves <https://github.com/brave/brave-browser/issues/37257>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and navigate to the `Portfolio` screen
2. Open the `Wallet Settings` menu, you should now see the `Portfolio Settings` in there.
3. Click on the the `Plus` button in the `Portfolio` header, it should open the `Edit Visible Assets` modal

https://github.com/brave/brave-core/assets/40611140/d7bb083f-963b-4539-9b6f-f5a02c8abf21
